### PR TITLE
Add jobs for 4.11 for interop tests

### PR DESCRIPTION
**Pull request by rabin-io**
This PR adds a 4.11 deployment with OCS tests

### CODE REVIEW

**Comment by rabin-io** [comment]
/pj-rehearse

Testing new image: ocs-ci-tests

**Comment by rabin-io** [comment]
/pj-rehearse

Set the logs folder for OCS test logs

**Comment by rabin-io** [comment]
/pj-rehearse

fix the path for cluster dir

**Comment by rabin-io** [comment]
/pj-rehearse

rewrite cnv-tests image

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml [hunk 1]
```diff
@@ -34,6 +42,8 @@ tests:
     test:
     - ref: interop-tests-deploy-cnv
     - ref: interop-tests-deploy-odf
+    - ref: cucushift-installer-check-cluster-health
```
Comment (liangxia) [inline ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml:45]: Here should be `- chain: cucushift-installer-check-cluster-health`

**Comment by rabin-io** [comment]
/pj-rehearse

dockerfile_literal

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml (hunk 1 above)
Comment (rabin-io) [reply ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml]: thx, I changed the line

**Comment by rabin-io** [comment]
/pj-rehearse

after the images were mirrored

**Comment by rabin-io** [comment]
/pj-rehearse

after the images were mirrored

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-main-ocp-cnv-odf-4.11-cnv-odf-tests-aws-upi-ocp411

**Comment by rabin-io** [comment]
/pj-rehearse

final

**Comment by rabin-io** [comment]
Hi all  (@myakove , @dshchedr, @liangxia)

This PR need yours attention :)
I think it is ready to be merged now.

**Comment by rabin-io** [comment]
Hi all  (@myakove, @dshchedr, @liangxia)

This PR need yours attention :)
I think it is ready to be merged now.

**Comment by rabin-io** [comment]
Hi all  (@myakove, @dshchedr, @liangxia)

This PR need yours attention :)
I think it is ready to be merged now.

File: ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh [hunk 2]
```diff
@@ -4,165 +4,41 @@ set -o nounset
...
+
+FINISH_TIME=$(date "+%s")
+DIFF_TIME=$((FINISH_TIME-START_TIME))
+set +x
+
+if [[ ${DIFF_TIME} -le 600 ]]; then
+    echo ""
+    echo " 🚨  The tests finished too quickly (took only: ${DIFF_TIME} sec), pausing here to give us time to debug"
+    echo "  😴 😴 😴"
+    sleep 7200
```
Comment (liangxia) [inline ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh:40]: Looks like this line is used for debug ?

Comment (rabin-io) [reply ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh:40]: Yes, this should give us 2h to do some investigation, without the need to reprovision the cluster again, just to verify it and catch it just at the right time.

Comment (rabin-io) [reply ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh:40]: @liangxia Can we resolve this thread and merge the PR?

Comment (liangxia) [reply ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh:40]: Ack

**Pull request merged**